scripts/schema_to_enums_v201.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+import sys
+import re
+import json
+from pathlib import Path
+
+
+def create_class(name):
+    return normalclass(name)
```

Comment:
I'm not sure why this function exists. Why not call the constructor of `normalclass` directly?

Reply:
Legacy code from the other script